src/test/resources/json/getconsignment_query_alldata.json

```diff
@@ -1 +1 @@
-{"query":"{getConsignment(consignmentid: \"b130e097-2edc-4e67-a7e9-5364a09ae9cb\"){seriesid, consignmentid, userid, totalFiles, fileChecks {antivirusProgress {filesProcessed}, checksumProgress {filesProcessed}, ffidProgress {filesProcessed}}, parentFolder}}"}
\ No newline at end of file
+{"query":"{getConsignment(consignmentid: \"b130e097-2edc-4e67-a7e9-5364a09ae9cb\") {seriesid, consignmentid, userid, totalFiles, fileChecks {antivirusProgress {filesProcessed}, checksumProgress {filesProcessed}, ffidProgress {filesProcessed}}, parentFolder, series {name}, transferringBody {name}}}"}
```

Comment:
Does the frontend use the code rather than the name in the series drop-down? If so, I think it's the code that we should be depending on here.

Reply:
Ah, yes, you are correct it's the code; I confused it with the Body